src/Packagist/WebBundle/Controller/WebController.php

```diff
@@ -38,16 +39,17 @@ public function indexAction()
 
     public function searchFormAction(Request $req)
     {
-        $form = $this->createForm(new SearchQueryType, new SearchQuery);
+        $form = $this->createForm(SearchQueryType::class, new SearchQuery(), [
+            'action' => $this->generateUrl('search.ajax'),
+            'method' => 'GET',
+        ]);
 
         $filteredOrderBys = $this->getFilteredOrderedBys($req);
         $normalizedOrderBys = $this->getNormalizedOrderBys($filteredOrderBys);
 
         $this->computeSearchQuery($req, $filteredOrderBys);
 
-        if ($req->query->has('search_query')) {
-            $form->submit($req);
-        }
+        $form->handleRequest($req);
```

Comment:
I think the last time we tried to change this we had to specifically revert it because the conditional check was necessary for the search page to function properly.